content/en/blog/news/2022-gsoc-work-report.md

```diff
@@ -0,0 +1,69 @@
+---
+title: "Jenkins X building a new UI"
+date: 2022-11-13
+draft: false
+description: This project replaces the older UI with the new UI with more powerful features.
+categories: [blog]
+keywords: [Community, jenkins-x, gsoc, 2022]
+slug: "GSoC-2022-work-report"
+aliases: []
+author: Rajat Gupta
+---
+
+# Jenkins X
+Jenkins X is an open-source solution that provides automated continuous integration and continuous delivery (CI/CD) and automated testing tools for cloud-native applications on Kubernetes.
+
+## Why need a new UI?
+A good UI is an essential for a CI/CD tool, as not everyone is familiar with the CLI.
+
+The current UI (jx-pipeline-visualizer) is a read-only UI, user can view the logs of PipelineActivity but can not start nor stop the pipeline.
+
+Features that UI will provide: 
+- Start and Stop a PipelineActivity.
+- Have a audit trail.
+- A DAG representation of PipelineActivity.
+- RBAC to prevent the distribution of cluster access.
+
+New Jenkins X UI focus on Simplicity, Security and a Superb User Experience.
+
+This is NOT GA(General Availability) yet.
+
+Visit the project repo [here](https://github.com/jenkins-x/jx-ui) to try it.
+
+## Work Done
+### Stop a running or pending PipelineActivity
+
+To stop release and pull request builds we can run:
+```bash
+jx pipeline stop
+```
+
+It would be much more convenient to have this functionality in the UI instead (Every CI/CD tool in the market has this functionality, so it's not something new)
+
+**Issue:** https://github.com/jenkins-x/jx-ui/issues/28
+
+**PRs:**
+- https://github.com/jenkins-x/jx-ui/pull/50
+- https://github.com/jenkins-x/jx-ui/pull/51
+- https://github.com/jenkins-x/jx-ui/pull/59
+- https://github.com/jenkins-x/jx-ui/pull/75
+
+
+### Show message with status of the PipelineActivity
```

Comment:
Is this done? I not remove it, and describe more about what you did. Write something about Svelte and how you have a golong backend and a Svelte fronted serverd in the same container

Reply:
It is done in the backend but still needs to be added in the frontend.

> Write something about Svelte and how you have a golong backend and a Svelte fronted serverd in the same container

I was thinking of adding that in the intro section.